<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: rgba(50, 50, 50, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>81</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_model_move">
       <property name="text">
        <string>Move</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_mmove_y">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_mmove_x">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_mmove_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_mmove_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_mmove_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_mmove_z">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>81</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_mrotate">
       <property name="text">
        <string>Rotate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_mrotate_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_mrotate_y">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_mrotate_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_mrotate_x">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_mrotate_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_mrotate_z">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>580</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lights">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>600</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Light source</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>630</y>
      <width>81</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lbl_lmove">
       <property name="text">
        <string>Move</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_lmove_y">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_lmove_x">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_lmove_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_lmove_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_lmove_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_lmove_z">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="lbl_model_scale">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_mscale_x">
         <property name="text">
          <string>K:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_mscale_x">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_lapply">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>680</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
   }</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lcancel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>710</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
   }</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_power">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>630</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_power">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>650</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="virus_count_label">
           <property name="text">
            <string>Количество частиц</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="virus_count_field"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="speed_label">
         <property name="text">
          <string>Скорость</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="speed_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>190</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
    }</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
    }</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_loadScene">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
    }</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
    }</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_mapply">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
   }</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mcancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 150);
    border-radius: 5px;
    background: rgba(20, 20, 20, 150);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 150);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 150);
   }</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>330</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>250</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="walls_label">
           <property name="text">
            <string>Стены</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="walls_comboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="floor_label">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="floor_comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="changeColorCheckBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Менять цвет при оседании</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
